packages/slice-machine/lib/models/common/Environment.ts

```diff
@@ -23,7 +22,7 @@ export interface FrontEndEnvironment {
   intercomHash?: string;
   manifest: Models.Manifest;
   repo: string;
-  changelog: PackageChangelog;
+  changelog?: PackageChangelog;
```

Comment:
how come it's optional now?

Reply:
Because the change log will be undefined on first render. We need to wait for the API call to the changelog/ endpoint for it to be set.